src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -474,19 +475,17 @@ public boolean exec(String code, boolean blocking) {
       } else {
         interp.exec(code);
       }
-
-      // FIXME - TOO MANY DIFFERENT CODE-PATHS TO interp.exec ...
-      // FIXME - FOR EXAMPLE - SHOULDN"T THERE BE AN
-      // INVOKE(finishedExecutingScript) !!! HERE ???
-
       return true;
-
     } catch (PyException pe) {
       // something specific with a python error
       error(pe.toString());
       invoke("publishStdError", pe.toString());
     } catch (Exception e) {
       error(e);
+    } finally {
+      if (blocking) {
+        invoke("finishedExecutingScript");

```

Comment:
nice.. maybe we can actually have proper gesture started / gesture stopped support in inmoov now.

Reply:
mebbe .. or at least a step closer - still needs unique mappings